src/main/java/io/jenkins/plugins/synopsys/security/scan/extension/pipeline/SecurityScanStep.java

```diff
@@ -35,6 +35,7 @@ public class SecurityScanStep extends Step implements Serializable {
     private transient String blackduck_token;
     private String blackduck_install_directory;
     private Boolean blackduck_scan_full;
+    private Boolean blackduckIntelligentScan;
```

Comment:
This variable shouldn't be in this class as it is mapped with `Jenkinsfile`. Therefor, it will create problems in future.